### DISCUSSION TITLE
Twitter OAuth2

### DISCUSSION THREAD

**Discussion by agusmdev** [fastapi-users/fastapi-users#945] [Q&A] [answered]
Do you have an easy way to integrate Twitter OAuth2? I've just found out that you have several providers but not Twitter :/
Do you have any suggestions? I'm trying to build a service where you can log in with your Twitter account and perform some automated interactions in Twitter

**Comment by frankie567** [comment, marked as answer]
See my answer here: https://github.com/frankie567/httpx-oauth/issues/14#issuecomment-1074972445
